playbooks/roles/ocp_operator_deployment/tasks/main.yml

```diff
@@ -0,0 +1,37 @@
+---
+# tasks file for ocp_operator_deployment
+- name: Verify that required variables are present
+  ansible.builtin.assert:
+    that:
+      - ocp_operator_deployment_operators is defined
+      - ocp_operator_deployment_operators | length > 0
+      - ocp_operator_deployment_version is defined
```

Comment:
Can't we install just the vatalog without installing operators?

Reply:
The name of the role is operator deployment. Catalog itself just required infrastructure. If we simply need to deploy catalogSource then better to use redhatci.ocp.catalog_source role.